client/reader/post-time/index.jsx

```diff
@@ -24,9 +26,11 @@ var PostTime = React.createClass( {
 		ticker.off( 'tick', this._update );
 	},
 
+	smartSetState: smartSetState,
+
 	_update: function( date ) {
 		date = date || this.props.date;
-		this.setState( { ago: humanDate( date ) } );
+		this.smartSetState( { ago: humanDate( date ) } );
```

Comment:
so, PRM doesn't prevent the all the work that happens just to get to `shouldComponentUpdate`. setting state enqueues work for the component, whether the new state is different or not. This avoids that work when we don't need to do it.